README.md

```diff
@@ -17,7 +17,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - name: Checkout
-      uses: actions/checkout@v1
+      uses: actions/checkout@v2
```

Comment:
> Checkout actions 2 has been released. So update all examples.

Have you tested this with v2? The reason the readme specifically uses `v1` is because there were some conflicting issues in the nightly releases.

Reply:
This major update is due to improved fetch performance and the dropped submodule support. The rest of the behavior is exactly the same, and you only benefit from improved fetch performance in common use cases.

Reply:
See also: https://github.com/actions/checkout/releases/tag/v2.0.0

Reply:
I tested this and it doesn’t work, originally the action referenced @master but it never ended up populating the changes to Github Pages. I think the build will technically pass if you add a ‘git pull’ in the Init function, but you won’t see a page build triggered so your changes will never go live.

Reply:
My guess is it’s related to the token config persistence. It’s likely that the GITHUB_TOKEN variable is being included automatically by the action which is causing some funky behavior. I can investigate this more when I get home.